### ISSUE TITLE
Reproduction of results for ESOL and FreeSolv

### ISSUE THREAD

**Issue by subhalingamd** [IBM/molformer#9]
Hi, thanks for releasing the pre-trained model and the code. Could you share the scripts used for fine-tuning on ESOL and FreeSolv data?

I am more interested in the hyper-parameters. I made the scripts similar to the Lipophilicity script but got way higher RMSE (e.g., more than 1 in case of FreeSolv).

Thanks.

**Comment by GintasKam** [comment]
could the RMSE's in the paper have been computed on the standardized values rather than the original ones?.. I think that was also the issue in another (BARTSmiles) llm paper that showed order-of-magnitude improvements in regression tasks. 

for example, in the MolFormer repositories' data the lipophilicity values seem to be standardized (centered around 0 and all with ~10 decimal points) whereas the MoleculeNet datasets are in the 0-7 range and fewer decimal points. clarification around the regression datasets' treatment would be very appreciated!

**Comment by lian-xiao** [comment]
Did you fix this？I tried to replicate the results of these two datasets using the normalized results, and although RMSE has decreased, it still falls short of the indicators reported in the paper.
If you have solved the problem, I hope you can tell me the solution, thank you very much

**Comment by maxall41** [comment]
I think the paper's results are for MolFormer-XL, not the published weights, which are just for the model trained on 10% of the data.
